### PR TITLE
feat: form components and refactoring

### PR DESCRIPTION
**Pull request by mortennordseth**
This PR touches multiple things, making it a bit hard to read, and I apologize for that. The PR: 
- introduces a error component
- includes new form components (textarea, radio, select)
- refactor to use the Typo component
- remove guard for TransportMode as it didn't work properly

### CODE REVIEW

File: src/page-modules/contact/components/input/input.tsx [hunk 1]
```diff
@@ -27,27 +29,22 @@ export const Input = ({
...
+      <label>
+        <Typo.span textType="body__primary">{t(label)}</Typo.span>
+      </label>
       <input
         type={type}
         name={name}
         className={andIf({
           [style.input]: true,
           [style.input__error]: errorMessage ? true : false,
+          [style.input__radio]: type === 'radio',
```
Comment (jonasbrunvoll) [inline src/page-modules/contact/components/input/input.tsx:41]: Should be removed

File: src/page-modules/contact/components/input/select.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  placeholder,
+}: SelectProps<T>) {
+  const id = useId();
+  const showError = !!error;
+
+  const findItemFromOptions = (id: string) =>
+    options.find((opt) => valueToId(opt) == id);
+
+  const onChangeInternal = (val: string) => {
+    console.log(val);
```
Comment (jonasbrunvoll) [inline src/page-modules/contact/components/input/select.tsx:35]: Remove console.logs

File: src/page-modules/contact/components/input/input.tsx (hunk 1 above)
Comment (mortennordseth) [reply src/page-modules/contact/components/input/input.tsx]: Done

**Review by jonasbrunvoll — APPROVED** [review]
Remove the last console.logs. Otherwise, looks good!

**Pull request merged**